vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MapView.java

```diff
@@ -28,6 +28,12 @@ public MapView() {
                 new Coordinate(1233058.1696443919, 6351912.406929109));
         map.getFeatureLayer().addFeature(nurembergMarker);
 
+        map.addFeatureClickListener(e -> {
```

Comment:
Is it necessary to have this in this page?

Reply:
For me that page was just some sort of playground. Actually I think we can now remove it, or at least replace it with a couple of demos. Would suggest to ignore it for now and then replace it with demos in a separate PR.